libraries/standard/http/utest/http_utest.c

```diff
@@ -157,6 +257,16 @@ static void addRangeToExpectedHeaders( _headers_t * expectedHeaders,
 /* ============================ UNITY FIXTURES ============================== */
 void setUp( void )
 {
+    testResponse.pBuffer = ( uint8_t * ) &pTestResponse[ 0 ];
+    testResponse.bufferLen = strlen( pTestResponse );
+
+    /* Configure the http_parser mocks with their callbacks. */
+    http_parser_init_AddCallback( parserInitExpectationCb );
+    http_parser_settings_init_AddCallback( parserSettingsInitExpectationCb );
+    http_parser_execute_AddCallback( parserExecuteExpectationsCb );
+
+    /* Ignore the calls to http_errno_description. */
+    http_errno_description_IgnoreAndReturn( "Mocked HTTP Parser Status" );
 }
 
 /* called before each testcase */
```

Comment:
I think oscar's PR is fixing these comments right?

Reply:
Yes